plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/weighers/ExpectedTypeWeigher.kt

```diff
@@ -17,8 +17,8 @@ import org.jetbrains.kotlin.psi.UserDataProperty
 
 internal object ExpectedTypeWeigher {
     object Weigher : LookupElementWeigher(WEIGHER_ID) {
-        override fun weigh(element: LookupElement): Comparable<*>? =
-            element.matchesExpectedType
+        override fun weigh(element: LookupElement): MatchesExpectedType =
+            element.matchesExpectedType ?: MatchesExpectedType.NON_TYPABLE
```

Comment:
Any tests for this change?

Reply:
It's hard to test individual weighers by isolation. The effect of this change is covered in existing tests after some other weighers are added.